graphjet-core/src/test/java/com/twitter/graphjet/algorithms/counting/TopSecondDegreeByCountTest.java

```diff
@@ -159,7 +164,7 @@ public void testTopSecondDegreeByCountWithSocialProofTypeUnionsWithSmallGraph()
       new RequestedSetFilter(new NullStatsReceiver())
     ));
 
-    TopSecondDegreeByCountRequest topSecondDegreeByCountRequest = new TopSecondDegreeByCountRequest(
+    TopSecondDegreeByCountRequestForTweet topSecondDegreeByCountRequestForTweet = new TopSecondDegreeByCountRequestForTweet(
```

Comment:
For the record, I *hate* checkstyle because it's so damn inflexible. Sometimes breaking formatting runs makes code more readable. A classic example is something like:

```
a = a very long assignment
b = another very long assignment
c = yet another very long assignment
...
```

If any of those lines are longer than n, checkstyle will *force* you to break line, even though it breaks the logic of a group of assignments.